ping.go

```diff
@@ -134,6 +134,9 @@ type Pinger struct {
 	// Number of packets received
 	PacketsRecv int
 
+	// Number of duplicate packets received
+	PacketsRecvDuplicates int
```

Comment:
Should we make the counters explicit uint64 going forward? Or maybe make it a breaking change later?

Reply:
I thought about that, but PacketsRecv and others right above it are already ints, so it seemed weird to have ints and then one uint64

Reply:
I think I too would go for consistency here. Not opposed to changing them all at once elsewhere but it seems a bit out of scope for this PR.

Reply:
Consistency for now is fine.